src/Model/Scope/Condition.php

```diff
@@ -137,16 +139,23 @@ public function __construct($key, $operator = null, $value = null)
         }
     }
 
+    protected function setSystem($system = true)
```

Comment:
no, also not in Scope, this is horrible name!

I think we should have this method in Model only